### ISSUE TITLE
[BUG] Notification title does not change when using playlist

### ISSUE THREAD

**Issue by nicejhkim** [jhomlala/betterplayer#438]
**Describe the bug**
The title (also author and image) is not changed when the next datasource in playlist is played. It remains the first.

**To Reproduce**
1. Make a playlist which has several datasources with different notification titles.
2. Start play(video 1).
3. Notification shows with 'TITLE1'
4. Skip to video 2
5. Notification title should be changed to 'TITLE2'

**Example code*
setupData() of /example/lib/pages/playlist_page.dart
```

Future<List<BetterPlayerDataSource>> setupData() async {
    _dataSourceList.add(
      BetterPlayerDataSource(
        BetterPlayerDataSourceType.network,
        Constants.forBiggerBlazesUrl,
        notificationConfiguration: BetterPlayerNotificationConfiguration(
          showNotification: true,
          title: "TITLE1",
          author: "AUTHOR1",
          imageUrl: Constants.catImageUrl,
        ),
        subtitles: BetterPlayerSubtitlesSource.single(
          type: BetterPlayerSubtitlesSourceType.file,
          url: await Utils.getFileUrl(Constants.fileExampleSubtitlesUrl),
        ),
        placeholder: Image.network(
          Constants.catImageUrl,
          fit: BoxFit.cover,
        ),
      ),
    );
    _dataSourceList.add(
      BetterPlayerDataSource(
        BetterPlayerDataSourceType.network,
        Constants.bugBuckBunnyVideoUrl,
        notificationConfiguration: BetterPlayerNotificationConfiguration(
          showNotification: true,
          title: "TITLE2",
          author: "AUTHOR2",
          imageUrl: Constants.catImageUrl,
        ),
        placeholder: Image.network(
          Constants.catImageUrl,
          fit: BoxFit.cover,
        ),
      ),
    );
    _dataSourceList.add(
      BetterPlayerDataSource(
        BetterPlayerDataSourceType.network,
        Constants.forBiggerJoyridesVideoUrl,
        notificationConfiguration: BetterPlayerNotificationConfiguration(
          showNotification: true,
          title: "TITLE3",
          author: "AUTHOR3",
          imageUrl: Constants.catImageUrl,
        ),
      ),
    );
    return _dataSourceList;
  }
```

**Expected behavior**
Notification title should be changed to the title of the video currently played.

**Screenshots**

**Flutter doctor**
[✓] Flutter (Channel stable, 2.0.4, on macOS 11.1 20C69 darwin-x64, locale ko-KR)
[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.55.0)
[✓] Connected device (2 available)

**Better Player version**
- Version: 0.0.64

**Smartphone (please complete the following information):**
 - Device: Android Pixel4, Samsung SM G950N
 - OS: Android 11

**Comment by jhomlala** [comment]
@nicejhkim This will be fixed with next release.

**Comment by jhomlala** [comment]
Fixed in 0.0.65.
